torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2645,7 +2645,8 @@ Error PyTorchModelLoader::loadMean(const torch::jit::Node *ptNode) {
     }
   }
 
-  if (hasGlowIValueForValue(inputs[MeanInputs::keepdims])) {
+  if (inputs.size() > 2 &&
```

Comment:
What is this for?

Reply:
It's the good thing about having asan enabled by default internally. It flushed out this read overflow issue.